test/gristWebDriverUtils.ts

```diff
@@ -359,10 +362,16 @@ public async sendCommand(name: string, argument: any = null) {
     await this.waitForServer();
   }
 
-  public async focusOnWidget(widgetName: string|RegExp) {
-      await this.driver.findContentWait('.test-widget-title-text', widgetName, 5000).click();
-      await this.driver.findWait(".test-widget-title-cancel", 1000).click();
-      await this.waitForServer();
+  public async focusOnWidget(widgetName: string | RegExp) {
```

Comment:
What is this? It looks wrong. `findContent` will throw error if element is not present I think.
What does ctrl+O does? If it is correct it should also work on macs (with mod key).

Reply:
`isPresent()` should just return false is element is not present. 
CTRL+O is shortcut to focus on next widget.